docs/swagger/post.yaml

```diff
@@ -159,6 +159,54 @@ paths:
         '404':
           description: Post not found
 
+  '/post/{id}/upload-files':
```

Comment:
Is this still relevant?

Reply:
This doc is used for the endpoint:

- `['pattern' => 'post/<id:\d+>/upload-files', 'route' => 'rest/post/post/attach-files', 'verb' => 'POST'],`

If we will remove the base action `BaseContentController::actionAttachFiles($id)` and keep only `ContentController::actionAttachFiles($id)` then I will remove this doc.